sdcm/tester.py

```diff
@@ -762,6 +757,33 @@ def clean_resources(self):
         if self.monitors is not None:
             self.monitors.get_backtraces()
             self.monitors.download_monitor_data()
+
+            # upload prometheus data
+            prometheus_folder = glob.glob(os.path.join(self.logdir, '*monitor*'))[0]
```

Comment:
ok, will use only inner prometheus folder but code should be:
prometheus_folder = glob.glob(os.path.join(self.logdir, '*monitor*/*monitor*/prometheus/'))[0]